collections/src/main/java/com/couchbase/javaclient/DocOperations.java

```diff
@@ -62,7 +55,9 @@ public static void main(String[] args) {
 		parser.addArgument("-dt", "--template").setDefault("Person").help("JSON document template");
 		parser.addArgument("-de", "--expiry").type(Integer.class).setDefault(0).help("Document expiry in seconds");
 		parser.addArgument("-ds", "--size").type(Integer.class).setDefault(500).help("Document size in bytes");
-
+		parser.addArgument("-st", "--start").type(Integer.class).setDefault(0).help("Starting documents operations index");
```

Comment:
Do you want to call this range instead of index? index makes it look like it is niql related.

Reply:
Yep, start and end parameters are added for compatibility with all the other python data generators. Just to avoid multiple if...else... blocks in python code.
Compatibility in terms of function names and input parameters for these functions gives me a possibility to instantiate appropriate data generator in single place and then just use it through all the test suite